swagger.yaml

```diff
@@ -1377,6 +1377,41 @@ paths:
         "500":
           $ref: "#/components/responses/internal_server_error"
 
+  /txs/{hash}/redeemers:
```

Comment:
It would be better to add this section after metadata CBOR, not between metadata and metadata CBOR.

![image](https://user-images.githubusercontent.com/791309/131529661-3490484b-4879-4abf-be3c-18c2010a15cb.png)